app/views/spree/shared/_login.html.erb

```diff
@@ -1,36 +1,69 @@
-  <%= form_for Spree.user_class.new, as: :spree_user, url: spree_login_path do |f| %>
+  <%#= form_for Spree.user_class.new, as: :spree_user, url: spree_login_path do |f| %>
     <div id="password-credentials" class="mb-4">
       <div class="form-group mb-4">
         <p class="form-group checkout-content-inner-field">
-          <%= f.email_field :email, class: 'spree-flat-input', placeholder: Spree.t(:email), value: params.dig(:spree_user, :email) %>
+          <%#= f.email_field :email, class: 'spree-flat-input', placeholder: Spree.t(:email), value: params.dig(:spree_user, :email) %>
         </p>
       </div>
       <div class="form-group mb-4">
         <p class="form-group checkout-content-inner-field">
-          <%= f.password_field :password, class: 'spree-flat-input', placeholder: Spree.t(:password) %>
+          <%#= f.password_field :password, class: 'spree-flat-input', placeholder: Spree.t(:password) %>
         </p>
       </div>
     </div>
 
-    <% if Spree::Frontend::Config[:remember_me_enabled] %>
+    <%#if Spree::Frontend::Config[:remember_me_enabled] %>
       <div class="spree-checkbox-container mt-4 mt-sm-5 mt-lg-4 spree-mb-medium">
         <div class="spree-checkbox">
-          <%= f.check_box :remember_me %>
-          <%= f.label :remember_me, '' %>
+          <%#= f.check_box :remember_me %>
+          <%#= f.label :remember_me, '' %>
         </div>
 
-        <%= f.label :remember_me, Spree.t(:remember_me), class: 'spree-checkbox-label' %>
+        <%#= f.label :remember_me, Spree.t(:remember_me), class: 'spree-checkbox-label' %>
       </div>
-    <% end %>
+    <%# end %>
 
-    <%= f.submit Spree.t(:login), class: 'btn btn-primary btn-block spree-btn mt-2' %>
-  <% end %>
-
-  <% if spree.respond_to?(:recover_password_path) %>
+    <%#= f.submit Spree.t(:login), class: 'btn btn-primary btn-block spree-btn mt-2' %>
+  <%# end %>
+  <%# if spree.respond_to?(:recover_password_path) %>
     <div class="text-center forgot-password mb-4">
-      <%= link_to Spree.t(:forgot_password), spree.recover_password_path %>
+      <%#= link_to Spree.t(:forgot_password), spree.recover_password_path %>
     </div>
+  <%# end %>
+</div> -->
+
+
+<%= form_for Spree.user_class.new, as: :spree_user, url: spree_login_path do |f| %>
+  <nav class="breadcrumb-wrap d-flex flex-wrap align-items-center justify-content-between" id="password-credentials">
+    <ol class="breadcrumb">
+      <li class="breadcrumb-item"><a href="#">Home</a></li>
+      <li class="breadcrumb-item active">Sign In</li>
+    </ol>
+  </nav>
+  <h1 class="title">Sign In</h1>
+  <div class="signin-note">Welcome to our new site. If you had an account on our previous system, please reset your password to access the new system by click 
+    <%= link_to  spree.recover_password_path do%>
+      Forgot your password.
+    <%end%>
+    Contact us if you need further assistance.
+  </div>
+  <div class="form-group form-floating has-icon mb-20" >
```

Comment:
Please add an empty div on email/password field and add this hook into that div
id="password-credentials"